### PR TITLE
LITE-30797 Adjustments in autocomplete

### PR DESCRIPTION
**Pull request by paulinaczybir**

### CODE REVIEW

File: components/src/widgets/select/widget.vue [hunk 1]
```diff
@@ -168,6 +168,16 @@ const getDisplayText = (item) => {
   if (props.optionTextFn) return props.optionTextFn(item);
   return item[props.propText];
 };
+
+watch(
+  () => model,
+  (newValue) => {
+    /* eslint-disable-next-line */
+    console.log('watch in select', newValue.value);
```
Comment (nerimartinez) [inline components/src/widgets/select/widget.vue:176]: please remove this line

File: components/src/widgets/select/widget.vue [hunk 2]
```diff
@@ -168,6 +168,16 @@ const getDisplayText = (item) => {
   if (props.optionTextFn) return props.optionTextFn(item);
   return item[props.propText];
 };
+
+watch(
+  () => model,
```
Comment (nerimartinez) [inline components/src/widgets/select/widget.vue:173]: do you need a function here?

File: components/src/widgets/select/widget.spec.js [hunk 3]
```diff
@@ -154,4 +155,38 @@ describe('Select', () => {
...
+    it('should update selectedOption when model changes', async () => {
+
+    // Initial check
+    expect(wrapper.vm.selectedOption).toEqual({ id: '1', value: 'Option 1' });
+    
+    // Simulate prop update to trigger the watch
+    await wrapper.setProps({ modelValue: '2' });
+
+    /* eslint-disable-next-line */
+    console.log('model value', modelValue);
```
Comment (nerimartinez) [inline components/src/widgets/select/widget.spec.js:183]: remove this line

File: components/src/widgets/select/widget.vue (hunk 2 above)
Comment (paulinaczybir) [reply components/src/widgets/select/widget.vue]: Fixed

**Review by arnaugiralt — APPROVED** [review]

**Pull request merged**